tt_eager/tt_lib/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 
+if(not WITH_PYTHON_BINDINGS)
+    message(FATAL_ERROR "Remove include of tt_lib when WITH_PYTHON_BINDINGS flag is OFF")
+endif()
```

Comment:
I think you meant `if(NOT WITH_PYTHON_BINDINGS)`? Else it errors on my dev machine (CMake 3.29).

```
CMake Error at tt_eager/tt_lib/CMakeLists.txt:2 (if):
  if given arguments:

    "not" "WITH_PYTHON_BINDINGS"

  Unknown arguments specified
```